src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -33,13 +33,13 @@
     AbstractPreparer,
     SingleValueReplacer)
 from azure.core.exceptions import HttpResponseError
-from ..._client_factory import cf_mysql_flexible_private_dns_zone_suffix_operations, cf_postgres_flexible_private_dns_zone_suffix_operations
-from ...flexible_server_virtual_network import prepare_private_network, prepare_private_dns_zone, prepare_public_network, DEFAULT_VNET_ADDRESS_PREFIX, DEFAULT_SUBNET_ADDRESS_PREFIX
-from ...flexible_server_custom_postgres import DbContext as PostgresDbContext
-from ...flexible_server_custom_mysql import DbContext as MysqlDbContext
-from ...flexible_server_custom_mysql import _determine_iops
-from ..._flexible_server_util import get_mysql_list_skus_info
-from ..._util import retryable_method
+from azure.cli.command_modules.rdbms._client_factory import cf_mysql_flexible_private_dns_zone_suffix_operations, cf_postgres_flexible_private_dns_zone_suffix_operations
+from azure.cli.command_modules.rdbms.flexible_server_virtual_network import prepare_private_network, prepare_private_dns_zone, prepare_public_network, DEFAULT_VNET_ADDRESS_PREFIX, DEFAULT_SUBNET_ADDRESS_PREFIX
+from azure.cli.command_modules.rdbms.flexible_server_custom_postgres import DbContext as PostgresDbContext
+from azure.cli.command_modules.rdbms.flexible_server_custom_mysql import DbContext as MysqlDbContext
+from azure.cli.command_modules.rdbms.flexible_server_custom_mysql import _determine_iops
+from azure.cli.command_modules.rdbms._flexible_server_util import get_mysql_list_skus_info
+from azure.cli.command_modules.rdbms._util import retryable_method
```

Comment:
When import statement is `from ..._util import retryable_method`, this error is raised when run pytest: `Running: /opt/az/bin/python3 -m pytest -x -v --forked -p no:warnings --log-level=WARN -n auto --junit-xml /azure_cli_test_result/rdbms.xml --pyargs azure.cli.command_modules.rdbms`
```
________ ERROR collecting tests/latest/test_rdbms_flexible_commands.py _________
ImportError while importing test module '/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/az/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py:36: in <module>
    from ..._client_factory import cf_mysql_flexible_private_dns_zone_suffix_operations, cf_postgres_flexible_private_dns_zone_suffix_operations
E   ImportError: attempted relative import beyond top-level package
```

The `__name__` is `tests.latest.test_rdbms_flexible_commands` when run the above command, so `...` is beyond top-level package.
The `__name__` is `azure.cli.command_modules.rdbms.tests.latest.test_rdbms_flexible_commands` when run `azdev test rdbms` and it works.

Reply:
It fails at `TestDebPackages` Task.
https://github.com/Azure/azure-cli/blob/1c31408ecd5af9a47071fd80bac0764e4d3aef7f/scripts/release/debian/test_deb_package.py#L13-L21

Yes, we should not use`...`.